catchpoint-mixin/panels.libsonnet

```diff
@@ -0,0 +1,418 @@
+          targets=[t.bottomAvgRequestSuccessRatioTestName],
+          description='The lowest average success request ratio among all test names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withMax(1)
+        + g.panel.timeSeries.standardOptions.withMin(0)
+        + g.panel.timeSeries.standardOptions.withUnit('percentunit'),
+
+      bottomAvgRequestSuccessRatioNodeName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Bottom average success request ratio by nodes',
+          targets=[t.bottomAvgRequestSuccessRatioNodeName],
+          description='The lowest average success request ratio among all node names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withMax(1)
+        + g.panel.timeSeries.standardOptions.withMin(0)
+        + g.panel.timeSeries.standardOptions.withUnit('percentunit'),
+
+      topAvgConnectionSetupTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average connection setup time by tests',
+          targets=[t.topAvgConnectionSetupTimeTestName],
+          description='The top average connection setup time among all test names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgConnectionSetupTimeNodeName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average connection setup time by nodes',
+          targets=[t.topAvgConnectionSetupTimeNodeName],
+          description='The top average connection setup time among all node names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgContentLoadingTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average content loading time',
+          targets=[t.topAvgContentLoadingTimeTestName],
+          description='The top average content loading time among all test names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgContentLoadingTimeNodeName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average content loading time by nodes',
+          targets=[t.topAvgContentLoadingTimeNodeName],
+          description='The top average content loading time among all node names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
```

Comment:
There's an issue with the legend leaving trailing dashes in this panel:
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/e23c8461-f5f8-4272-b8ff-7f9e2c49431c)